.github/workflows/zeebe-ci.yml

```diff
@@ -156,8 +157,6 @@ jobs:
           -P skip-random-tests,parallel-tests,extract-flaky-tests
           verify
           | tee "${BUILD_OUTPUT_FILE_PATH}"
-      - name: Normalize artifact name
-        run: echo "ARTIFACT_NAME=$(echo ${{ matrix.project }} | sed 's/\//-/g')" >> $GITHUB_ENV
```

Comment:
:question: Why is this required?

Reply:
This step is a leftover, I removed it as such. The `ARTIFACT_NAME ` env will be always empty (there is no matrix defined, `${{ matrix.project }}`).